xml/System/ReadOnlySpan`1.xml

```diff
@@ -395,8 +424,10 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Returns a value that indicates the current read-only span is empty.</summary>
+        <value>
+          <format type="text/markdown"><![CDATA[`true` when the current span is empty; otherwise `false`.]]></format>
```

Comment:
The true and false keywords aren't formatted correctly.
![image](https://user-images.githubusercontent.com/6527137/42907298-409580e8-8a92-11e8-9638-6626783a329f.png)

Reply:
don't use markdown here otherwise IntelliSense will also be broken, since they're generated from the docs. you need to use the language keyword tag.